test/routing/ArrivalsRouterSpec.scala

```diff
@@ -24,6 +24,9 @@ import org.scalatest.matchers.must.Matchers
 import org.scalatestplus.mockito.MockitoSugar
 import play.api.http.HeaderNames
 import play.api.http.Status.ACCEPTED
+
+import play.api.http.Status.BAD_REQUEST
+
```

Comment:
Very much a nitpick, but can we make sure that imports like this don't have newlines between them for non-Java/Scala imports?

Reply:
done